code.go

```diff
@@ -7,1387 +7,1405 @@ package multicodec
+	// Dccp is tagged "multiaddr" and described by: draft.
 	Dccp Code = 0x21 // dccp
 
-	// Murmur3_128 is tagged "multihash".
+	// Murmur3_128 is tagged "multihash" and described by: draft.
 	Murmur3_128 Code = 0x22 // murmur3-128
 
-	// Murmur3_32 is tagged "multihash".
+	// Murmur3_32 is tagged "multihash" and described by: draft.
 	Murmur3_32 Code = 0x23 // murmur3-32
 
-	// Ip6 is tagged "multiaddr".
+	// Ip6 is tagged "multiaddr" and described by: permanent.
 	Ip6 Code = 0x29 // ip6
 
-	// Ip6zone is tagged "multiaddr".
+	// Ip6zone is tagged "multiaddr" and described by: draft.
 	Ip6zone Code = 0x2a // ip6zone
 
-	// Path is tagged "namespace" and described by: Namespace for string paths. Corresponds to `/` in ASCII..
+	// Path is tagged "namespace" and described by: permanent.
 	Path Code = 0x2f // path
 
-	// Multicodec is tagged "multiformat".
+	// Multicodec is tagged "multiformat" and described by: draft.
 	Multicodec Code = 0x30 // multicodec
 
-	// Multihash is tagged "multiformat".
+	// Multihash is tagged "multiformat" and described by: draft.
 	Multihash Code = 0x31 // multihash
 
-	// Multiaddr is tagged "multiformat".
+	// Multiaddr is tagged "multiformat" and described by: draft.
 	Multiaddr Code = 0x32 // multiaddr
 
-	// Multibase is tagged "multiformat".
+	// Multibase is tagged "multiformat" and described by: draft.
 	Multibase Code = 0x33 // multibase
 
-	// Dns is tagged "multiaddr".
+	// Dns is tagged "multiaddr" and described by: permanent.
 	Dns Code = 0x35 // dns
 
-	// Dns4 is tagged "multiaddr".
+	// Dns4 is tagged "multiaddr" and described by: permanent.
 	Dns4 Code = 0x36 // dns4
 
-	// Dns6 is tagged "multiaddr".
+	// Dns6 is tagged "multiaddr" and described by: permanent.
 	Dns6 Code = 0x37 // dns6
 
-	// Dnsaddr is tagged "multiaddr".
+	// Dnsaddr is tagged "multiaddr" and described by: permanent.
 	Dnsaddr Code = 0x38 // dnsaddr
 
-	// Protobuf is tagged "serialization" and described by: Protocol Buffers.
+	// Protobuf is tagged "serialization" and described by: draft.
 	Protobuf Code = 0x50 // protobuf
 
-	// Cbor is tagged "serialization" and described by: CBOR.
+	// Cbor is tagged "serialization" and described by: permanent.
 	Cbor Code = 0x51 // cbor
 
-	// Raw is tagged "ipld" and described by: raw binary.
+	// Raw is tagged "ipld" and described by: permanent.
 	Raw Code = 0x55 // raw
 
-	// DblSha2_256 is tagged "multihash".
+	// DblSha2_256 is tagged "multihash" and described by: draft.
 	DblSha2_256 Code = 0x56 // dbl-sha2-256
 
-	// Rlp is tagged "serialization" and described by: recursive length prefix.
+	// Rlp is tagged "serialization" and described by: draft.
 	Rlp Code = 0x60 // rlp
 
-	// Bencode is tagged "serialization" and described by: bencode.
+	// Bencode is tagged "serialization" and described by: draft.
 	Bencode Code = 0x63 // bencode
 
-	// DagPb is tagged "ipld" and described by: MerkleDAG protobuf.
+	// DagPb is tagged "ipld" and described by: permanent.
```

Comment:
surely these are all regressions? the old description was better

Reply:
I am with you. Maybe something to improve in `go generate` or the CSV structure?